File: sum.rb
p arr = (1..100).select{|item| item % 2 == 0}.map{|item| item * item}

p arr.select{|item| item > 50}.inject{|sum, n| sum + n}

p (1..12).inject(1){|a, item| a * item}

def col(n)
  if n%2 == 0 then
    n/2
  else
    n*3+1
  end
end

File: sum.c
#include<stdio.h>
#define N 100

int main()
{
  int i, sum=0, arr[N];

  for(i=0; i<N; i++)
    {
      arr[i] = i+1;
    }

  for(i=0; i<N; i++)
    {
      sum += arr[i];
    }

  printf("%d\n", sum);
    

  return 0;
}
